pykrige/ok.py

```diff
@@ -404,6 +404,34 @@ def display_variogram_model(self):
                                         self.lags), 'k-')
         plt.show()
 
+    def get_variogram_points(self):
+        """Obtain the data pairs for the variogram model.
+           returns a numpy.array with the form [lag, variogram_value_at_lag]

```

Comment:
The type of return variable(s) should be documented see e.g. https://github.com/bsmurphy/PyKrige/blob/4c54e28da59dccd3620e586cf38e88f56a4c5eca/pykrige/ok.py#L556

Reply:
I will add docstrings for that